ios/CountlyTestApp-iOS/AppDelegate.m

```diff
@@ -32,11 +32,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     config.host = @"https://YOUR_COUNTLY_SERVER";
     config.enableDebug = YES;
 
-//    config.features = @[CLYPushNotifications, CLYCrashReporting, CLYAutoViewTracking];     //Optional features
+//    config.features = @[CLYFeature.pushNotifications, CLYCrashReporting, CLYAutoViewTracking];     //Optional features
```

Comment:
I think for Objective C we don't need to change it

Reply:
done